CMS-Enterprise/easi-app
# Easi 4645/send emails

# EASI-4645

## Description

- connect resolvers to sending emails

## How to test this change

- create a system intake locally
- add a user of your choice as a GRB reviewer
- open postman (`EASI/System Intake/GRB Meeting (Async)/Create GRB Discussion Group Tags`)
- enter the created system intake id as well as the user's uuid in the tag span
- send the request
- confirm an email goes out

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
